src/components/controls/Dropdown.tsx

```diff
@@ -133,11 +144,10 @@ class Dropdown extends React.Component<IDropdownProps, IDropdownState> {
                                 options={displayedItems}
                                 isDisabled={props.disabled}
                                 isSearchable={true}
+                                filterOption={null}
                                 placeholder={'Valitse...'}
                                 styles={_getCustomStyles(this.props)}
-                                noOptionsMessage={() =>
-                                    props.isNoOptionsMessageHidden ? null : 'Ei hakutuloksia'
-                                }
+                                noOptionsMessage={() => null}
```

Comment:
Why?

Reply:
Forgot to change it back after debugging. Though after testing I feel like it could be better like this.

The message serves no purpose. When there is no search results to be shown the dropdown options is closed and the input validation message "Lyhyttunnus on jo käytössä" can be seen which is basically the same thing as saying "Ei hakutuloksia". Also if the user types in an actual validation error (i.e bad format, too long) and the validation error is hidden under the "Ei hakutuloksia" message.

Reply:
How about other dropdowns? All dropdowns are search enabled by default.

Reply:
Also if this is really always defined as `noOptionsMessage={() => null}` then current props isNoOptionsMessageHidden serves no purpose and should be removed. I would revert this change though.

Reply:
Reverted